pkg/logql/limits.go

```diff
@@ -14,6 +14,7 @@ var (
 // Limits allow the engine to fetch limits for a given users.
 type Limits interface {
 	MaxQuerySeries(userID string) int
+	MaxQueryRange(userID string) time.Duration
```

Comment:
let's call it `MaxQueryInterval`.
otherwise, we add the same confusion as we have on Grafana UI . range <-> interval

Reply:
Ok, I was wrong )))) this limit applies to `[range_value]` not to the whole query range .... then let's leave it as is ;)